CMSgov/onemac
# OY2-21899 fix: Kristin can't spell - Concurrence !

Story: https://qmacbis.atlassian.net/browse/OY2-21899
Endpoint: See github-actions bot comment

### Details
Fix Concurrance to be the correct spelling: Concurrence

### Changes
- Constant name updated: PENDING_CONCURRANCE to PENDING_CONCURRENCE
- Update status text "Pending - Concurrance" to "Pending - Concurrence"

### Test Plan
1. Login as a CMS user
2. Verify that the Status reads: Pending - Concurrence on the package dashboard
